sklearn/linear_model/least_angle.py

```diff
@@ -1145,10 +1154,14 @@ def fit(self, X, y):
         # Now compute the full model
         # it will call a lasso internally when self if LassoLarsCV
         # as self.method == 'lasso'
-        Lars.fit(self, X, y)
+        Lars._fit(self, X, y, max_iter=self.max_iter, alpha=best_alpha,
+                  Xy=None, fit_path=True)
         return self
 
+    # XXX deprecate?
     @property
+    @deprecated("Attribute alpha is deprecated in 0.18 and "
+                "will be removed in 0.20. See 'alpha_' instead")
```

Comment:
this was a hack to be able to call Lars.fit but the CV classes should have an alpha_ but no alpha param